README.rst

```diff
@@ -53,6 +53,8 @@ The required dependencies to use the software are:
 * Nibabel >= 3.0
 * Pandas >= 1.0
 
+For a comprehensive list of dependencies, see the file ``nilearn/setup.cfg``.
```

Comment:
I wonder if  we should remove the min versions from the readme to avoid having to update them in 2 places? if we put a link to the setup.cfg that may be enough

Reply:
That sounds reasonable. I'll do that.